src/components/Admin/WikiDetailsCards.tsx

```diff
@@ -80,9 +80,9 @@ export const WikiDetailsCards = ({
               {weeklyValue} this week
             </Text>
           </VStack>
-          <CircularProgress value={percentage} color={color} size="45px">
+          <CircularProgress value={percentage || 0} color={color} size="45px">
```

Comment:
``CircularProgressValue = percentage || 0 ``, kindly replace the ``percentage || 0`` anywhere it's used.

```suggestion
          <CircularProgress value={CircularProgressValue} color={color} size="45px">
```